src/components/PageSettingsModal/PageSettingsSchema.jsx

```diff
@@ -56,12 +57,13 @@ export const PageSettingsSchema = (existingTitlesArray = []) =>
       }
     }),
     layout: Yup.string().oneOf(["file", "post", "link"]),
-    date: Yup.date()
+    date: Yup.string()
```

Comment:
Hmm am confused here... wouldn't using the `Yup.date()` be a better solution than using a string and then trying to manually match the regex pattern ourself?

Reply:
Hmm we're expecting a specific date string here(`yyyy-mm-dd`) to be saved in the frontmatter of the file - using `Yup.date()` specifically specifies it as a Date object, which we need to transform back into a string for the purposes of validation and to save - I opted to change it back to string because of this!